lib/Logger/LogAppenderFile.cpp

```diff
@@ -263,7 +264,8 @@ void LogAppenderFile::reopenAll() {
 #endif
 
     if (!Logger::_keepLogRotate) {
-      FileUtils::remove(backup);
+      // TODO Is it acceptable here to ignore failures?
```

Comment:
IMO it is not great if we can't remove the old, closed logfile here, but it is not catastrophic. We can't easily log from this position in the code, so I guess it is acceptable to leave this place as it is.